### PR TITLE
Added recipe for chartkick

### PR DESCRIPTION
**Pull request by pmlandwehr**
I'm pushing a recipe for `0.4.2` instead of `0.5` because [`airflow`](https://pypi.python.org/pypi/airflow) has a pinned dependency that I’d like to get right. Once airflow is building, we should be able to tick the [`chartkick`](https://pypi.python.org/pypi/chartkick/0.4.2) version correspondingly. We can wrangle with `airflow` the next time it updates.

(Pinging @mher in case they'd like to be added as a maintainer to the `chartkick` builder on conda-forge, a library of community-build [`conda`](http://conda.pydata.org) packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how the package is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/chartkick-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that's entirely fine too.)

### CODE REVIEW

File: recipes/chartkick/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  version: "0.4.2"
+
+test:
+  imports:
+    - chartkick
+    - chartkick.templatetags
+
+about:
+  home: https://github.com/mher/chartkick.py
+  license: MIT License
```
Comment (jakirkham) [inline recipes/chartkick/meta.yaml]: Just "MIT" is fine. Can drop redundant "License".

File: recipes/chartkick/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+package:
+  name: chartkick
+  version: "0.4.2"
```
Comment (jakirkham) [inline recipes/chartkick/meta.yaml]: This is a bit weird. Is this some copy-pasta?

Comment (pmlandwehr) [inline recipes/chartkick/meta.yaml]: Oh whoops. Yes. Part skeleton, part template. Funny that it built.

**Comment by jakirkham** [comment]
LGTM

**Pull request merged**